Woof/Woof/View/SitterFlow/SitterProfileView/SitterProfileView.swift

```diff
@@ -45,24 +51,35 @@ struct SitterProfileView: View {
                         ratePerHour: viewModel.pricePerHour
                     )
 
-                    Button("Edit") {
+                    Button(editButtonLabelText) {
                         isEditingMode.toggle()
                     }
-                    .disabled(viewModel.name.isEmpty)
                 }
                 .buttonStyle(CapsuleWithWhiteText())
                 .padding()
                 .background(Color.App.purpleLight)
                 .cornerRadius(AppStyle.UIElementConstant.cornerRadius)
             }
+
             Spacer()
         }
         .padding(.horizontal)
     }
 
     // MARK: - Private interface
 
+    private let cancelButtonLabelText = "Cancel"
+    private let saveButtonLabelText = "Save"
+    private let editButtonLabelText = "Edit"
     @State private var isEditingMode = false
+
+    private func cancelEditing() {
+        viewModel.name = viewModel.currentSitter.name
+        viewModel.surname = viewModel.currentSitter.surname
+        viewModel.phone = viewModel.currentSitter.phone
+        viewModel.bio = viewModel.currentSitter.bio
+        viewModel.pricePerHour = viewModel.currentSitter.pricePerHour.description
```

Comment:
I think you should move this logic to the view model.

Reply:
You're right, thanks.
Fixed it.